x-pack/plugins/security_solution/common/endpoint/schema/actions.ts

```diff
@@ -139,3 +139,13 @@ export const EndpointActionFileDownloadSchema = {
 export type EndpointActionFileDownloadParams = TypeOf<
   typeof EndpointActionFileDownloadSchema.params
 >;
+
+/** Schema that validates the file info API */
+export const EndpointActionFileInfoSchema = {
+  params: schema.object({
+    action_id: schema.string({ minLength: 1 }),
+    agent_id: schema.string({ minLength: 1 }),
+  }),
+};
+
```

Comment:
fair. It would probably need a new name, but they're both referencing the same concept (a file uploaded through the action) so presumably the same inputs will be used to get-info on it and download it -- into the future

Reply:
Yeah, I thought about using the Download one, but held off on it and just cloned it. I prefer for the schemas to match the names of the API where they are used  - for clarity. I could also just:

```
export const EndpointActionFileInfoSchema = EndpointActionFileDownloadSchema;
```

Will add a test file (maybe in next PR)

Reply:
Yeah, I agree with the schema name matching the API and also thought of what you suggested here with the download schema assigned to a named const.

Reply:
I'll leave it as is for now and will think about changing it when I add a test for it